ci-operator/step-registry/origin/e2e-aws-ovn-hybrid/origin-e2e-aws-ovn-hybrid-workflow.yaml

```diff
@@ -10,5 +10,6 @@ workflow:
     - ref: origin-e2e-test
     post:
     - chain: ipi-deprovision
+    - chain: gather-network
```

Comment:
@pecameron the test failures now are failure to talk to the server. I don't think we can try to grab logs from everywhere *after* we've torn the cluster down. Swap the order of these two? (note that ipi-deprovision chain runs 'gather' before everything else so that's a precedent for this)

Reply:
changed